toolbox/linux/how.rst

```diff
@@ -2,9 +2,11 @@ OK I'm Ready... How Do I Try/Install It?
 ========================================
 
 So you're ready to install a Linux? Great! If it's your first time, hopefully
-you're starting with Ubuntu. Also if it's your first time, you may want to grab
-an experienced Linux buddy (ask in Slack or mailing list) to help you. Don't do
-this if you know you'll desperately need your computer the next day!
+you're starting with Ubuntu. Also if it's your first time, you may want the help
+of an experienced Linux buddy. If you don't know where to find one, you could
+ask in Slack or on the ``hackers-discuss`` mailing list (more info here:
+:doc:`/organizational/communications`). Don't do this if you know you'll
```

Comment:
Unfortunately, this can now be read as "Don't ping slack/mailing list if you know you'll desperately need your computer the next day!"